docs/index.yaml

```diff
@@ -1,8 +1,18 @@
 apiVersion: v1
 entries:
+  andi:
+  - apiVersion: v1
+    appVersion: "1.0"
+    created: 2019-07-16T14:41:51.46638071-04:00
+    description: REST API to allow for dataset definition ingestion
+    digest: 87a558e09dc5ee07fbe8182060aa40e43d2ed1d6055280007cbc372a6d7f1a56
+    name: andi
+    urls:
+    - andi-1.0.0.tgz
+    version: 1.0.0
```

Comment:
Do we want to inc this version?

Reply:
This is the first version of this chart so far as the outside world is concerned. Version 1.0 seems appropriate.